README.md

```diff
@@ -119,6 +119,96 @@ If you want to further configure each individual server you can look into:
 https://github.com/input-output-hk/cardano-node/blob/master/nix/nixos/cardano-node-service.nix#L136
 to see all the options available for configuration.
 
+### Cardano Tracer
+
+After deploying you can setup `cardano-tracer` to monitor the nodes with RTView or
+Prometheus and EKG servers (Please see more [here](https://github.com/input-output-hk/cardano-node/tree/master/cardano-tracer#readme))
+
+Currently this deployment repository is configured to work with cardano-tracer and
+monitoring the nodes via RTView (please check the specific [RTView page](https://github.com/input-output-hk/cardano-node/blob/master/cardano-tracer/docs/cardano-rtview.md)).
+Succintly there are 2 ways of running RTView:
+
+- Local
+- Distributed
+
+In the local way, `cardano-tracer` and `cardano-node` instances all run in the same
+machine. In the distributed way `cardano-tracer` runs on one machine while the nodes
+run on different machines.
+
+The one we want is the distributed one, where `cardano-tracer` runs on the deployer
+machine and the nodes on the deployed AWS instances.
+
+There are two particular things about the distributed way of setting up `cardano-tracer`:
+it can act as a Server or as a Client. As a Server, `cardano-tracer` waits for connections
+from `cardano-node` instances (so nodes are clients); and as a Client, `cardano-tracer`
+connects to `cardano-node` instances (so nodes are servers). We pick which way we want by
+configuring `cardano-tracer` with an `AcceptAt` or `ConnectTo` attribute and running the
+nodes with a `--tracer-socket-connect` or `--tracer-socket-accept`.
+
+For our particular situation, we are going to use `cardano-tracer` as a client and
+`cardano-node` instances as servers, since we can only establish an one-way connection
```

Comment:
```suggestion
For our particular situation, we are going to use `cardano-tracer` as a client and
`cardano-node` instances as servers, since we can only establish a one-way connection
```